ding/framework/supervisor.py

```diff
@@ -243,7 +243,7 @@ def _recv_queue(self, queue: Union[queue.Queue, mp.Queue]):
     def start_link(self) -> None:
         if not self._running:
             for child in self._children:
-                child.start(recv_queue=self._recv_queue)
+                child.start(self._recv_queue)
```

Comment:
why modify this

Reply:
Sorry, I accidentally modified it when I was doing the test, and it has been reset~